designs.js

```diff
@@ -67,35 +37,27 @@ function makeList(){
 
 }
 
-function deleteRow(event){
-  const delButton = event.target;
-  const rowIndex = delButton.parentElement.parentElement.rowIndex;
-  listOfItems.deleteRow(rowIndex);
-  event.stopPropagation();
+function deleteRow(){
+  if(selectedRowIndex > 0){
+    listOfItems.deleteRow(selectedRowIndex);
+    selectedRowIndex = 0;
+  }
 }
 
-function upRow(event){
-  const rowToMove = event.target.parentElement.parentElement;
-  const table = rowToMove.parentElement;
-  if(rowToMove.rowIndex > 1){
-    if(selectedRowIndex == rowToMove.rowIndex){
-      selectedRowIndex--;
-    }
-    const refRow = table.rows[rowToMove.rowIndex-1];
-    table.insertBefore(rowToMove, refRow);
+function upRow(){
+    if(selectedRowIndex > 1){
```

Comment:
Could you please fix indent here and in line #57 as well?

Reply:
I have done it